docs/tbdev_getting_started.ipynb

```diff
@@ -79,7 +79,8 @@
       "outputs": [],
```

Comment:
<p>I think this can be a clearer. Can you rephrase as:</p><p> </p><p>TensorBoard logs are created during training by passing the TensorBoard (link) and hyperparameters callbacks (link) to Keras' Model.fit(). These logs can then be uploaded to TensorBoard.dev.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done. Do you think "https://www.tensorflow.org/tensorboard/get_started" is the correct "TensorBoard" link?